ci/build_container/build_recipes/protobuf.sh

```diff
@@ -2,16 +2,12 @@
 
 set -e
 
-# Unless overriden with an explicit release tag, e.g. v3.2.0rc2, we use a pinned
-# HEAD commit. This is only until we get a release with
-# https://github.com/google/protobuf/pull/3327, i.e. v3.4.0.
-[ -z "$ENVOY_PROTOBUF_COMMIT" ] && ENVOY_PROTOBUF_COMMIT=062df3d0724d9ae5e3c65d481dc1d3aca811152e  # 2017-07-20
```

Comment:
We can revert this change then, it's trivial. Git has history, so I don't think there is any point in leaving unused code / hacks just in case we might need them someday.